metricbeat/module/windows/_meta/config.yml

```diff
@@ -0,0 +1,13 @@
+- module: windows
+  metricsets: ["perfmon"]
+  enabled: true
+  period: 1s
+  perfmon.counters:
```

Comment:
It's great to have these example coutners which can be used. The way we handled it in the jmx metricset is that we defined only `perfmon.counters:` but left it empty (https://github.com/elastic/beats/blob/master/metricbeat/module/jolokia/_meta/config.yml#L8) and move the examples to the docs: https://github.com/elastic/beats/blob/master/metricbeat/module/jolokia/jmx/_meta/docs.asciidoc